arm-devtestlabs/2016-05-15/swagger/DTL.json

```diff
@@ -1,4124 +1,8639 @@
-                    },
-                    { "$ref": "#/parameters/api-version" }
-                ],
-                "responses": {
-                    "200": {
-                        "description": "OK",
-                        "schema": { "$ref": "#/definitions/Policy" }
-                    },
-                    "default": {
-                        "description": "BadRequest",
-                        "schema": { "$ref": "#/definitions/CloudError" }
-                    }
-                },
-                "security": [ { "oauth2": [ "user_impersonation" ] } ]
-            }
-        },
-        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DevTestLab/labs/{labName}/schedules": {
-            "get": {
-                "tags": [ "Schedule" ],
-                "description": "List schedules in a given lab.",
-                "operationId": "Schedule_List",
-                "consumes": [ ],
-                "produces": [ "application/json", "text/json" ],
-                "parameters": [
-                    { "$ref": "#/parameters/subscriptionId" },
-                    {
-                        "name": "resourceGroupName",
-                        "in": "path",
-                        "description": "The name of the resource group.",
-                        "required": true,
-                        "type": "string"
-                    },
-                    {
-                        "name": "labName",
-                        "in": "path",
-                        "description": "The name of the lab.",
-                        "required": true,
-                        "type": "string"
-                    },
-                    {
-                        "name": "$filter",
-                        "in": "query",
-                        "description": "The filter to apply on the operation.",
-                        "required": false,
-                        "type": "string"
-                    },
-                    {
-                        "name": "$top",
-                        "in": "query",
-                        "description": "The maximum number of resources to return from the operation.",
-                        "required": false,
-                        "type": "integer",
-                        "format": "int32"
-                    },
-                    {
-                        "name": "$orderBy",
-                        "in": "query",
-                        "description": "The ordering expression for the results, using OData notation.",
-                        "required": false,
-                        "type": "string"
-                    },
-                    { "$ref": "#/parameters/api-version" }
-                ],
-                "responses": {
-                    "200": {
-                        "description": "OK",
-                        "schema": { "$ref": "#/definiti
```

Comment:
`resourceGroupName` should be defined in global `parameters` section and reused as `subscriptionId`